### PR TITLE
AVRO-3715 Make the build work again.

### PR DESCRIPTION
**Pull request by nielsbasjes**
Fix python formatting problems.

### CODE REVIEW

File: build.sh [hunk 1]
```diff
@@ -42,7 +42,7 @@ change_java_version() {
 # This might not have been sourced if the entrypoint is not bash
 [[ -f "$HOME/.cargo/env" ]] && . "$HOME/.cargo/env"
 
-set -xe
+set -e
```
Comment (martin-g) [inline build.sh:45]: I find `-x` useful for debugging.

Comment (nielsbasjes) [reply build.sh:45]: Ok

**Comment by nielsbasjes** [comment]
@martin-g Can you please check if the type fix I did is correct. My python skills are too limited for me to know for sure if this is the correct way.

**Review by martin-g — APPROVED** [review]
I am not a Python person too but it looks good to me!

**Pull request merged**